jupyter.py

```diff
@@ -0,0 +1,8 @@
+"""Launch the root jupyter command
+
+Alias to jupyter_core
+"""
+
```

Comment:
I started the repo's version at 4.0 like everything else, but maybe I shouldn't. Should I start this one at 1.0, to make it clear that it doesn't have any relationship to the versions of the 'real' packages?